Assets/Scripts/Gameplay/GameplayObjects/PrefabSpawner.cs

```diff
@@ -0,0 +1,15 @@
+using UnityEngine;
```

Comment:
this should be in utils, like our other simple scripts (like auto destroy, things like that)